apps/wallet/src/ui/app/staking/stake/StakeForm.tsx

```diff
@@ -46,48 +49,111 @@ function StakeForm({
 
     const [formatted, symbol] = useFormatCoin(coinBalance, coinType);
 
+    const allCoins = useAppSelector(accountCoinsSelector);
+    const allCoinsOfTransferType = useMemo(
+        () => allCoins.filter((aCoin) => aCoin.type === coinType),
+        [allCoins, coinType]
+    );
+    const [coinDecimals] = useCoinDecimals(coinType);
+    const gasEstimationUnformatted = useMemo(() => {
```

Comment:
Probably most of the cases it will be a pay + the stake call right? In the end is will be 2 txs usually. Probably fine to keep it as is but maybe add a todo to improve it when we will have the new dry run

Reply:
Actually it might be better to use `DEFAULT_GAS_BUDGET_FOR_STAKE` as it is used from the validation